ml/ml_table.yaml

```
@@ -71,24 +71,31 @@ groups:
   data_type_inc: DynamicTable
   doc: A table for storing sequence data
   datasets:
-  - data_type_inc: TrainValidationTestMask
+  - name: tvt_mask
+    data_type_inc: TrainValidationTestMask
     doc: a column indicate if a sample was used for training, testing or validation
     quantity: '?'
-  - data_type_inc: CrossValidationMask
+  - name: cv_mask
+    data_type_inc: CrossValidationMask
     doc: a column to indicate which cross-validation fold a sample was part of
     quantity: '?'
-  - data_type_inc: ClassProbability
+  - name: class_prob
+    data_type_inc: ClassProbability
     doc: a column to store the class probability for each class across the samples
     quantity: '?'
-  - data_type_inc: ClassLabel
+  - name: class_label
+    data_type_inc: ClassLabel
     doc: a column to store which class a sample was classified as
     quantity: '?'
-  - data_type_inc: RegressionOutput
+  - name: reg_output
```

```suggestion
  - name: estimate
```

This I'm less certain of

---

How about prediction?

---

Yeah, I thought about that. But sometimes prediction gets used for class labels in classification